0002-capture-matching-next-steps.md

```diff
@@ -0,0 +1,115 @@
+# Selecting the next steps to take to improve capture matching capabilties
+
+* Status: proposed
+* Deciders: @ZavenArra @DavidEzraJay Cam Webb, @Elforama {list everyone involved in the decision using github handle} <!-- optional -->
+* Date: 2022-05-11
+
+Technical Story:
+* https://greenstand.gitbook.io/capture-matching/
+
+
+## Context and Problem Statement
+
+We have implemented a basic manual capturing tool that allows admin users to visually match repeat captures that are withing 10 meters of an initial capture.  This approach remains slow and difficult to do correctly.  We have identified a range of enhancements that will improve the usability of this tool, and need to decide where to focus resources.
+
+
+## Decision Drivers <!-- optional -->
+
+* Ease of implementing enhancements
+* Improvement to capture matching efficiency and accuracy offered by enhancement
+* Stability of enhancement across diverse use cases
+* Ease of use for affected users
+
+## Considered Options
+
+1. Implement mobile application side image collection improvements, including enforced phone rotation, enforced compass bearing, and perhaps others.
+2. Research feasibility of leveraging autocorrelation between GPS measurements to improve selection of candidates for location based matching of captures.
+3. Create experimental manual capture matching UI that offers manual spatial analysis relevant to disambiguating matching captures.
+4. Research approaches to image analysis that can produce a similarity metric to improve selection of candidates for location based matching of captures.
+5. Attempt to train models using machine learning based on existing database of capture images to identify repeat captures.
+6. Incorporate assigned species information during capture matching.
+7. Create curated training sets of matched trees for to support training of machine learning based models.
+<!--
+## Decision Outcome
+
+Chosen option: "{option 1}", because {justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force {force} | … | comes out best (see below)}.
+
+### Positive Consequences <!-- optional -->
+
+* {e.g., improvement of quality attribute satisfaction, follow-up decisions required, …}
+* …
+
+### Negative Consequences <!-- optional -->
+
+* {e.g., compromising quality attribute, follow-up decisions required, …}
+* …
+-->
+
+## Pros and Cons of the Options
+
+### 1. Implement mobile application side image collection improvements, including enforced phone rotation, enforced compass bearing, and perhaps others.
```

Comment:
Do we currently ask the mobile user if it's a first capture or a recapture, or do we only collect this information where orgs make use of the notes field?
I'm wondering if there are any more cues we can collect at the point of capture, without making the app more difficult to use.

Reply:
We do not currently ask if it is the first capture or not. One challenge our organizational users are having is that multi growers are claiming the same tree as the "first" capture. 

The original app had an "update" or "new tree" button. The update button gave the user the option of which tree they were updating based on GPS. The function worked well in small scale (10 or 20 trees) but any more than that and it just crashed.  

![Screenshot (59)](https://user-images.githubusercontent.com/25363572/170890543-37756955-1ade-4d61-a1f6-3ba19c2e5f11.png)

Reply:
My view is that we need to be able to verify user entered claims related to the tree update/species/date/claims etc

Reply:
When I reviewed this version of the app 4 years ago, I determined that the crashing issue was not the major obstacel to the implementation.  The main issues were:
1.  'update tree' cannot obtain a geofix accurate enough to determine which tree is being updated (the origin of the capture matching problem)
2. users do not necessarily know if they are updating or a tracking a new tree
3. downloading relevant data for a tree update process to the application requires data connectivity in the field, and appropriate bandwith.